api/notd/store/retriever.py

```diff
@@ -9,10 +9,10 @@
 from core.store.retriever import StringFieldFilter
 from sqlalchemy.sql.expression import func as sqlalchemyfunc
 
-from notd.model import Token
+from notd.model import Token, TokenMetadata
 from notd.model import TokenTransfer
-from notd.store.schema import TokenTransfersTable
-from notd.store.schema_conversions import token_transfer_from_row
+from notd.store.schema import TokenTransfersTable,TokenMetadataTable
+from notd.store.schema_conversions import token_transfer_from_row,token_metadata_from_row
```

Comment:
put new imports on new lines pls. you can run `isort --sl notd` in the api/ folder and it should do it for you